### ISSUE TITLE
Cannot convert a Symbol value to a string

### ISSUE THREAD

**Issue by dmytro-lymarenko** [shouldjs/should.js#87]
```
const destinations = {
	Europe: ['Amsterdam', 'London', 'Kiev'],
	America: ['New York', 'San Francisco']
};

sr = TestUtils.createRenderer();
sr.render(<SelectDestination destinations={destinations} />);
tree = sr.getRenderOutput();
```
`should(tree.props.children).be.Array().and.have.length(1);` - throw the error:
```
TypeError: Cannot convert a Symbol value to a string
      at Formatter.String (native)
      at Formatter.format (node_modules/should/node_modules/should-format/index.js:39:14)
      at Formatter.formatPropertyValue (node_modules/should/node_modules/should-format/index.js:146:23)
      at Formatter.<anonymous> (node_modules/should/node_modules/should-format/index.js:61:72)
      at Array.forEach (native)
      at Formatter._formatObject (node_modules/should/node_modules/should-format/index.js:56:14)
      at Formatter.<anonymous> (node_modules/should/node_modules/should-format/index.js:271:15)
      at Formatter.format (node_modules/should/node_modules/should-format/index.js:39:14)
      at Formatter.formatPropertyValue (node_modules/should/node_modules/should-format/index.js:146:23)
      at Formatter.<anonymous> (node_modules/should/node_modules/should-format/index.js:61:72)
      at Array.forEach (native)
      at Formatter._formatObject (node_modules/should/node_modules/should-format/index.js:56:14)
      at Formatter._format_object_array (node_modules/should/node_modules/should-format/index.js:287:15)
      at Formatter.format (node_modules/should/node_modules/should-format/index.js:39:14)
      at Object.exports.format (node_modules/should/lib/util.js:128:36)
      at Error.Object.create.generateMessage.value (node_modules/should/lib/assertion-error.js:75:25)
      at Error.AssertionError.Object.defineProperty.get (node_modules/should/lib/assertion-error.js:17:34)
      at Error.Object.create.generateMessage.value (node_modules/should/lib/assertion-error.js:73:29)
      at Error.AssertionError.Object.defineProperty.get (node_modules/should/lib/assertion-error.js:17:34)
      at Error.Object.create.generateMessage.value (node_modules/should/lib/assertion-error.js:73:29)
      at Error.AssertionError.Object.defineProperty.get (node_modules/should/lib/assertion-error.js:17:34)
      at Error.toString (native)
      at Function.prepareStackTrace (node_modules/babel/node_modules/babel-core/node_modules/source-map-support/source-map-support.js:303:16).
```
`console.log(tree.props.children);` output the following:
```
[ { '$$typeof': Symbol(react.element),
    type: 'div',
    key: null,
    ref: null,
    props: { className: 'citySelect', tabIndex: '-1', children: [Object] },
    _owner: null,
    _store: {} },
  { '$$typeof': Symbol(react.element),
    type: { [Function] displayName: 'DestinationsContainer' },
    key: null,
    ref: null,
    props: 
     { containerStyle: [Object],
       destinations: [Object],
       selectCity: [Object] },
    _owner: null,
    _store: {} } ]
```
when I check `should(tree.props.children).be.Array().and.have.length(2);` it works properly.
I also checked `['a', 'b'].should.have.length(1);` - it works properly.

**Comment by btd** [comment]
By stacktrace problem in formatter. To be sure where you have error (node/browser, which version) and which version of should.js you are using?

**Comment by btd** [comment]
I cannot reproduce this for now. Need more details:
```
var format = require('should-format');

var obj = [ { '$$typeof': Symbol.for('react.element'),
    type: 'div',
    key: null,
    ref: null,
    props: { className: 'citySelect', tabIndex: '-1' },
    _owner: null,
    _store: {} },
  { '$$typeof': Symbol.for('react.element'),
    key: null,
    ref: null,
    _owner: null,
    _store: {} }
  ]
console.log(format(obj))
```

returns withouts errors:
```
Array [
  Object {
    $$typeof: Symbol(react.element),
    _owner: null,
    _store: Object {},
    key: null,
    props: Object { className: 'citySelect', tabIndex: '-1' },
    ref: null,
    type: 'div'
  },
  Object {
    $$typeof: Symbol(react.element),
    _owner: null,
    _store: Object {},
    key: null,
    ref: null
  }
]
```

Show me full object using util.inspect(tree.props.children, { depth: null }) or explain how i can reproduce your error.

**Comment by dmytro-lymarenko** [comment]
"should": "^7.1.0"

**Comment by dmytro-lymarenko** [comment]
node --version
v0.12.7

**Comment by btd** [comment]
I just published 7.1.1 give it a try pls.

**Comment by dmytro-lymarenko** [comment]
Thanks. Now it works.
